hibernate-core/src/main/antlr/org/hibernate/grammars/graph/GraphLanguageParser.g4

```diff
@@ -33,21 +33,21 @@ attributeList
     ;
 
 attributeNode
-	: attributePath (subGraph)?
+	: attributePath subGraph?
 	;
 
 attributePath
-	: NAME attributeQualifier?
+	: ATTR_NAME attributeQualifier?
 	;
 
 attributeQualifier
-	: DOT NAME
+	: DOT ATTR_NAME
 	;
 
 subGraph
 	: LPAREN (subType COLON)? attributeList RPAREN
 	;
 
 subType
-	: NAME
+	: TYPE_NAME
```

Comment:
so the following errors flushed out

- uppercase should be allowed in NAME_CONTINUATION;
- subType should starts with uppercase;